src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -658,48 +516,63 @@ def _upload_and_run_pipeline(
             location=self.config.location,
         )
 
-        logger.info(
-            "Submitting pipeline job with job_id `%s` to Vertex AI Pipelines "
-            "service.",
-            job_id,
-        )
+        if self.config.workload_service_account:
+            logger.info(
+                "The Vertex AI Pipelines job workload will be executed "
+                "using the `%s` "
+                "service account.",
+                self.config.workload_service_account,
+            )
+        if self.config.network:
+            logger.info(
+                "The Vertex AI Pipelines job will be peered with the `%s` "
+                "network.",
+                self.config.network,
+            )
 
-        # Submit the job to Vertex AI Pipelines service.
         try:
-            if self.config.workload_service_account:
+            if schedule:
                 logger.info(
-                    "The Vertex AI Pipelines job workload will be executed "
-                    "using the `%s` "
-                    "service account.",
-                    self.config.workload_service_account,
+                    "Scheduling job using native Vertex AI Pipelines scheduling..."
+                )
+                run.create_schedule(
```

Comment:
might make sense to check for the following properties in schedule and giving a warning if they are set (Saying vertex only supports cron jobs)

start_time
end_time
interval_second
catchup

Reply:
@htahir1 in fact VertexAI supports `start_time` and `end_time` fields to define a window in which the Cron expression will be active (see https://cloud.google.com/vertex-ai/docs/pipelines/schedule-pipeline-run#rest)
Do you think it is worth implementing this?

Reply:
It's a nice addition / flexibility. If you can, I'd add this, yes.

Reply:
Yes i would add it @francoisserra

Reply:
@htahir1 : Resolved by [7b432ce](https://github.com/zenml-io/zenml/pull/2310/commits/7b432ce6ce65c2b20b7974f766a9214b7fd8ab7d)